### DISCUSSION TITLE
Lit-html + Angular: how to disable Dev Mode?

### DISCUSSION THREAD

**Discussion by rosostolato** [lit/lit#3260] [Q&A] [answered]
I have a project that I'm using Angular but in Ag-Grid cell renders I'm using the `lit-html` package to render templates. But I can't find a way to disable the "Dev Mode" and remove the warning in console:

![image](https://user-images.githubusercontent.com/27731641/187961464-ea657230-e7df-440c-a28f-37a41fc8c2b9.png)

How am I supposed to disable it?

**Comment by justinfagnani** [comment, marked as answer]
The production builds are the default, under the `"default"` package export condition. The dev builds are using the `"development"` export condition and have to be specifically opted-into.

It looks like you're using webpack-dev-server, which I presume is use the `"development"` export condition, which is _good_: you'll get extra warnings and more easily read stack traces.

In your production build your toolchain should be selecting the `"default"` package export condition automatically. Hopefully you don't see this warning in prod.

**Comment by rosostolato** [comment]
Sweet! I'll check if in prod it's OK. Thanks!
